tensorflow/compiler/xla/service/gpu/gpu_executable.h

```diff
@@ -133,10 +135,10 @@ class GpuExecutable : public Executable {
 
   const std::vector<uint8> binary_;
 
-  // The compute capability of the GPU. In the runtime, GpuExecutable check with
-  // either amd's isa version, or nvdia's major minor version, depending on the
-  // gpu hardware
-  absl::variant<std::pair<int, int>, int> compute_capability_;
+  // The GPU version. In the runtime, GpuExecutable check with
```

Comment:
comment need re-layout

Reply:
Do you mean I should update the comment? FYI formatting-wise I have run with clang-format before push.

Reply:
clang-format doesn't deal with comments if I understand correctly. Wording here also needs to be tuned. Also the paragraph is missing `.` in its end.

Reply:
Addressed in latest commit. 

BTW I played around with clang-format a little bit turns out that it does adjust comment as well.